src/communication/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+1.8.1
```

Comment:
Update this version to 1.8.1b1

Reply:
Thanks @anmolbohra97. I've updated the version in
        modified:   src/communication/HISTORY.rst
        modified:   src/communication/azext_communication/version.py

Reply:
Updated to 1.8.0b2 per comment from @jsntcy